docs/cli.md

```diff
@@ -94,10 +103,80 @@ With `--all` flag, you create config file including all default setting of offic
       executable_methods:
         - method_name: "node_governance_alarm"`
 ```
-
 Also, You can set the home directory of VATZ (default "~/.vatz") with `--home` flag.
 And you can create the new config file with your disired name with `--output` flag
 
+## Start
+Please, ensure completion of the configuration setup in either the `default.yaml` file or your custom configuration YAML file prior to initiating VATZ.
```

Comment:
Please ensure the configuration setup is completed in either the `default.yaml` file or your custom configuration YAML file before initiating VATZ.

Reply:
seems more clear!!